### DISCUSSION TITLE
Enhancement Request: 9:16 Shorts Rendering in React-Native-IFrame for Youtube Shorts

### DISCUSSION THREAD

**Discussion by bilalshams7** [LonelyCpp/react-native-youtube-iframe#329] [General]
Hi React-Native-IFrame Team,

Would it be possible to add functionality for rendering shorts in a 9:16 aspect ratio? This feature would greatly enhance the mobile viewing experience, allowing content to fully utilize the screen space.

Appreciate all the effort you put into making this tool better!

Best wishes,

Bilal

**Comment by BeMoreDifferent** [comment]
I created a workaround url for embedding: https://bemoredifferent.github.io/public-youtube-iframe/iframe.html?fullscreen=true to allow for fullscreen solution. I think this is not the perfect solution but it's working so if anybody wants to use it or integrate a improved version I would be happy. 

`baseUrlOverride="https://bemoredifferent.github.io/public-youtube-iframe/iframe.html?fullscreen=true"`

Cheers, Daniel
